.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This is for VisualStudioCode, yes? If so, please remove it, there should be no IDE/editor specific code in the library.